src/css/lessons.css

```diff
@@ -0,0 +1,113 @@
+.lessons-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.lessons-list {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  row-gap: 20px;
+}
+
+.lessons-list-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 335px;
+}
```

Comment:
Second list item must be green (currently implemented as a focus) by default (according to info provided by mentor on planning).